### DISCUSSION TITLE
Make Halloween Masks Tradeable

### DISCUSSION THREAD

**Discussion by Dextrator** [chromiecraft/chromiecraft#5847] [Suggestions]
Hello everyone,

I am an achievement hunter.
Everyone who has ever tried to go for 100% completion probably knows that in WotLK the most difficult part is not earning the loremaster, fishing or killing ultra rare creatures, but an total RNG event that you can only attempt once a year for 14 days.

The 24 Halloween Masks are only available by completing the daily dungeon that rewards 1 mask per day in WoTLK, but in Cataclysm Blizz got rid of this problem by creating an NPC that sells them for a few coppers. The masks are entirely worthless other than the cosmetic value as they give no buffs or armor. 

I understand it wouldn't be blizzlike, but I would love if you would make a small change and introduce either this NPC, or make the masks purchasable in some way, or at least tradeable so community can share it among themselves for the achievement.

**Comment by Hihilist** [comment]
If you are referring to https://www.wowhead.com/wotlk/achievement=284/a-mask-for-all-occasions then you need 20, not 24 masks and it has 4 possible sources:
- daily dungeon as you said (we never had it on CC so far),
- treat bag that can be obtained every 1 hour at Inn (you may be unlucky and get trick instead of treat),
- gift box after doing the daily quest with putting down the fires,
- a handful of candy items, obtained by visiting candies pumpkins in inns,
so while it is still RNG, it is more generous than you think and achievement is doable within 1-2 years if you are persistent.
 
(for reference, see https://wowgaming.altervista.org/aowow/?item=20565#contained-in-item)
